content/SCALE/DataProtection/Replication.md

```diff
@@ -0,0 +1,192 @@
+
+{{< expand "Process Summary" "v" >}}
+## Process Summary
+
+* **Tasks > Replication Tasks**
+  * Choose sources for snapshot replication.
+    * Remote sources require an SSH connection.
+    * TrueNAS shows how many snapshots will be replicated.
+  * Define the snapshot destination.
+    * A remote destination requires an SSH connection.
+    * Choose destination or define manually by typing a path.
+      * Adding a new name on the end of the path creates a new dataset.
+  * Choose replication security.
+    * We always recommend Replication with encryption.
+    * Disabling encryption is only meant for absolutely secure networks.
+  * Schedule the replication.
+    * Schedule can be standardized presets or a custom defined schedule.
+    * Running once runs the replication immediately after creation.
+      * Task is still saved and can be rerun or edited.
+  * Choose how long to keep the replicated snapshots.
+{{< /expand >}}
+
+## Creating a Remote Replication Task
+
+To create a new replication, go to **Data Protection > Replication Tasks** and click *ADD*.
+
+![TasksReplicationTasksAddSCALE](/images/SCALE/RepWhatWhereSCALE.png "Add new Replication Task")
+
+You can load any saved replication to prepopulate the wizard with that configuration.
+Saving changes to the configuration creates a new replication task without altering the task that was loaded into the wizard.
+This saves some time when creating multiple replication tasks between the same two systems.
+
+{{< tabs "Replication Task Steps" >}}
+{{< tab "Sources" >}}
+Start by configuring the replication sources.
+Sources are the datasets or zvols with snapshots to use for replication.
+Choosing a remote source requires selecting an SSH connection to that system.
+Expanding the directory browser shows the current datasets or zvols that are available for replication.
+You can select multiple sources or manually type the names into the field.
+
+TrueNAS shows how many snapshots are available for replication.
+We recommend you manually snapshot the sources or create a periodic snapshot task *before* creating the replication task.
+However, when the sources are on the local system and don't have any existing snapshots, TrueNAS can create a basic periodic snapshot task and snapshot the sources immediately before starting the replication. Enabling *Recursive* replicates all snapshots contained within the selected source dataset snapshots.
+
+![TasksReplicationTasksAddSourceSCALE](/images/SCALE/RepAddSourceSCALE.png "Choosing a Local Source")
+
+Local sources can also use a naming schema to identify any custom snapshots to include in the replication.
+Remote sources require entering a *snapshot naming schema* to identify the snapshots to replicate.
+A naming schema is a collection of [strftime](https://www.freebsd.org/cgi/man.cgi?query=strftime) time and date strings and any identifiers that a user might have added to the snapshot name.
```

Comment:
We probably need an example here.

There also is a fourth option: replicating snapshots with names matching a specific regex, https://github.com/truenas/zettarepl Ctrl-F, `name-regex`

Reply:
@BonnieF you can see examples of naming schemas by creating a periodic snapshot task then editing it and finding the value. A basic naming schema that is created with a generic replication task is `auto-%Y-%m-%d_%H-%M`
@themylogin is the regex option something that is configurable from the UI?